run.sh

```diff
@@ -71,6 +71,15 @@ for COMMAND in "docker" "docker-compose"; do
     command_exists "${COMMAND}"
 done
 
+# Check if platform is Raspberry Pi, to apply it's own configuration later
+# This is done since amd64 computers do not have the same hardware device
+#  for GPIO pins like the Pi does.
+case $(uname -m) in
```

Comment:
I got an error here:
```
./run.sh: line 79: syntax error near unexpected token `)'
./run.sh: line 79: `    aarch64)'
```
I would use:
```bash
case $(uname -m) in
    armv7l | aarch64)
        raspberrypi="true" 
        ;;
```

Reply:
Sure, I don't actually remember how to use cases in bash, I use them so little 😅